docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,153 @@
+---
+
+import DualRegion from "./img/dual-region.svg";
+
+## Introduction
+
+Camunda 8 is compatible with a dual-region setup under certain [limitations](#limitations). This allows Camunda 8 to run in a mix of active-active and active-passive setups, resulting in an overall **active-passive** setup. The following will explore the concept, limitations, and considerations.
+
+You should get yourself familiar with the topic, the [limitations](#limitations) of Camunda 8 but also general [considerations](#considerations) on operating a dual-region setup.
+
+## Disclaimer
+
+:::danger
+
+- Customers must develop and test operational procedures in non-production environments based on the framework steps outlined by Camunda.
+- Before advancing to production go-live, customers need to validate these procedures with Camunda.
+
+:::
+
+## Architecture
+
+<DualRegion />
+
+The current architecture consists of the following supported components:
+
+- Zeebe (Workflow Engine)
+- Elasticsearch (Database)
+- WebApps
+  - Operate
+  - Tasklist
+
+Zeebe stretches across the regions due to its use of the [raft protocol](<https://en.wikipedia.org/wiki/Raft_(algorithm)>) allowing it to communicate and replicate data between all brokers across. Zeebe is exporting data to two Elasticsearch instances, one in each region. Operate and Tasklist will import the previously exported data and run per region.
+
+One of the regions will be considered **active** and the other one **passive**. User traffic must only reach the **active** region.
+
+### Definitions
+
+**Active-active** and **active-passive** are standard setups used in dual region configurations to ensure that applications remain available and operational in case of failures.
+
+In an **active-active** setup, multiple application instances run simultaneously in different regions, actively handling user requests. This allows for better load balancing and fault tolerance, as traffic can spread across regions. If one region fails, the workload can shift to another without causing disruptions.
+
+In contrast, an **active-passive** setup designates one region as the main or active region where all user requests are processed. The other region remains on standby until needed, only becoming active if the primary region fails. This setup is easier to manage but may result in higher delays during failover events.
+
+### User traffic
+
+The overall system is **active-passive**, even though some components may be **active-active**. You will have to take care of the user traffic routing or DNS by yourself, and won't be considered further. You must select one region as the actively serving region and route the user traffic there. In case of a total region failure, you must route the traffic to the passive region yourself.
+
+<!-- Should we provide some reading materials on how to tackle this? -->
+
+### Components
+
+#### Zeebe ![active](https://img.shields.io/badge/active-green?style=for-the-badge)
```

Comment:
just a suggestion with shield.io (we would mirror it as local file to avoid injections if accepted).
Wanted to not having looking that triste.

Reply:
not sure about the badge as a single word, I think it should be `active-active` or `active-passive`, just `active` or `passive` on their own are confusing.

Something like this:
https://img.shields.io/badge/active/active-green?style=for-the-badge

Or just using a text instead of a badge.

Reply:
I removed it for now, the bold active-active / active-passive in each first sentence of the components indicates it quite quickly.
Was just an experiment.